### PR TITLE
Manage debezium and testcontainers on the top level to solve conflicts

### PR DESCRIPTION
**Pull request by ppalaga**
among participants

@gsmet We'd need something like this to make our debezium tests work.

### CODE REVIEW

**Comment by ppalaga** [comment]
The bottom line is that IMO testcontainers and other testing-only related dependencies should be banned from the platform BOM and it should be individual test participants' responsibility to add/manage them individually. In such a case, we should still check that those participant's test BOM do not override anything from the Platform BOM.

File: bom/runtime/pom.xml [hunk 1]
```diff
@@ -28,6 +28,51 @@
...
+                <artifactId>debezium-embedded</artifactId>
+                <version>${debezium.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>testcontainers-bom</artifactId>
+                <type>pom</type>
+                <scope>import</scope>
+                <version>1.14.3</version>
+            </dependency>
```
Comment (gsmet) [inline bom/runtime/pom.xml:73]: I'm not entirely sure this is a good idea to have it here given we decided to remove it from the Quarkus core BOM.

@aloubyansky what's your take on that one?

Comment (gsmet) [reply bom/runtime/pom.xml:73]: BTW, additional question: is it a test only dependency?

Comment (aloubyansky) [reply bom/runtime/pom.xml:73]: Whether it's only a test dep is the key question. Are they all necessary in the user-facing BOM @ppalaga ?

Comment (famod) [reply bom/runtime/pom.xml:73]: Just to make sure: Have you guys seen this: https://github.com/quarkusio/quarkus/issues/9968#issuecomment-650835627
and the resulting PR #81 here in quarkus-platform?

Comment (gsmet) [reply bom/runtime/pom.xml:73]: @famod thanks, I missed it and your analysis is indeed very useful. I'll try to come up with a contingency plan for 1.6 and a better plan for 1.7.

**Comment by gsmet** [comment]
Closing in favor of https://github.com/quarkusio/quarkus-platform/pull/92 .

**Pull request closed without merging**